From e29d1bbeae9f9e7bf98af7547ffa44bda779accf Mon Sep 17 00:00:00 2001
From: Yaroslav Bolyukin <iam@lach.pw>
Date: Sat, 18 Apr 2026 20:55:23 +0200
Subject: [PATCH 1/4] refactor: use mutable systemd

---
 qubes/vm/qubesvm.py | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/qubes/vm/qubesvm.py b/qubes/vm/qubesvm.py
index 9c8d67c..ebc6397 100644
--- a/qubes/vm/qubesvm.py
+++ b/qubes/vm/qubesvm.py
@@ -1268,18 +1268,17 @@ class QubesVM(qubes.vm.mix.net.NetVMMixin, qubes.vm.LocalVM):
                     "sudo",
                     "ln",
                     "-sf",
-                    "/usr/lib/systemd/system/qubes-vm@.service",
-                    "/etc/systemd/system/multi-user.target.wants/qubes-vm@"
+                    "/etc/systemd/system/qubes-vm@.service",
+                    "/etc/systemd-mutable/system/multi-user.target.wants/qubes-vm@"
                     "{}.service".format(self.name),
                 ]
             )
         else:
             retcode = subprocess.call(
                 [
-                    "sudo",
-                    "systemctl",
-                    "disable",
-                    "qubes-vm@{}.service".format(self.name),
+                    "rm",
+                    "-f",
+                    "/etc/systemd-mutable/system/system/multi-user.target.wants/qubes-vm@{}.service".format(self.name),
                 ]
             )
         if retcode:
-- 
2.53.0

