From e06c3b1bb3c30201c429659625e4067b3f665b98 Mon Sep 17 00:00:00 2001
From: Yaroslav Bolyukin <iam@lach.pw>
Date: Mon, 16 Sep 2024 18:54:00 +0200
Subject: [PATCH 1/4] refactor: use mutable systemd

Signed-off-by: Yaroslav Bolyukin <iam@lach.pw>
---
 qubes/vm/qubesvm.py | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/qubes/vm/qubesvm.py b/qubes/vm/qubesvm.py
index 8ce481d7..bce0d7fb 100644
--- a/qubes/vm/qubesvm.py
+++ b/qubes/vm/qubesvm.py
@@ -1177,18 +1177,17 @@ class QubesVM(qubes.vm.mix.net.NetVMMixin, qubes.vm.BaseVM):
                     "sudo",
                     "ln",
                     "-sf",
-                    "/usr/lib/systemd/system/qubes-vm@.service",
-                    "/etc/systemd/system/multi-user.target.wants/qubes-vm@"
+                    "/etc/systemd/system/qubes-vm@.service",
+                    "/etc/systemd-mutable/system/multi-user.target.wants/qubes-vm@"
                     "{}.service".format(self.name),
                 ]
             )
         else:
             retcode = subprocess.call(
                 [
-                    "sudo",
-                    "systemctl",
-                    "disable",
-                    "qubes-vm@{}.service".format(self.name),
+                    "rm",
+                    "-f",
+                    "/etc/systemd-mutable/system/system/multi-user.target.wants/qubes-vm@{}.service".format(self.name),
                 ]
             )
         if retcode:
-- 
2.47.1

